### PR TITLE
update gatsby-on-windows.md for nodejs install

### PR DESCRIPTION
**Pull request by memeree**
It isn't clear to people following the page for the first time how to install node git etc. 
scripts always seem to start with the assumption that the person has already installed nodejs on windows, but this isn't clear on the site around the steps to follow in order to do that. 
A good way on Windows is to install chocolatey which i've outlined, and then use that to install node etc.
just trying to help as a first time contributor.

Thanks
Moshin

### CODE REVIEW

File: docs/docs/gatsby-on-windows.md [hunk 1]
```diff
@@ -2,6 +2,40 @@
...
+Many of the scripts you will encounter will require you to use NodeJS and NPM on a Windows development environment; unfortunately this doesn't come as standard on Windows.
+
+A reliable way of installing all of these separate resources is to install Chocolatey https://chocolatey.org/
+Chocolatey is a package manager for Windows which helps install a lot of the software/dependencies listed in later tutourials and online. 
+
+Head on over to https://chocolatey.org/install to install the latest version for your windows environment.
+
+Installation instructions can be found for various tools on https://chocolatey.org/docs/installation
+
+Once installed, head on over to your Windows PowerShell (Admin) version to install Nodejs and git.
```
Comment (muescha) [inline docs/docs/gatsby-on-windows.md]: brand names
```suggestion
Once installed, head on over to your Windows PowerShell (Admin) version to install Node.js and Git.
```

File: docs/docs/gatsby-on-windows.md [hunk 2]
```diff
@@ -2,6 +2,40 @@
...
+
+```powershell
```
Comment (muescha) [inline docs/docs/gatsby-on-windows.md]: Just shell? ( https://www.gatsbyjs.org/contributing/gatsby-style-guide/#code-formatting-type-tab )
```suggestion
```shell
```

File: docs/docs/gatsby-on-windows.md [hunk 3]
```diff
@@ -2,6 +2,40 @@
...
+
+``` 
+
+NPM is bundled in with nodejs and so does not need to be separately installed.
+
+## Optional Code Editor : VSCode
+
+Assuming you want to use VSCode as an editor, you can do this also via Chocolatey using the following command:
+
+```powershell
```
Comment (muescha) [inline docs/docs/gatsby-on-windows.md]: same
````suggestion
```shell
````

**Comment by laurieontech** [comment]
Hey @memeree, thanks for the PR! We absolutely agree that we need to improve docs for non mac users. However, chocolately is a specific installation technique that we're not going to include at this time. 
If you'd like to participate in the work towards improving windows installation instructions we'd welcome it! This PR in particular is being verified and discussed at the moment. https://github.com/gatsbyjs/gatsby/pull/18262#pullrequestreview-308797493

**Pull request closed without merging**